app.rb

```diff
@@ -0,0 +1,122 @@
+require_relative 'person'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'classroom'
+require_relative 'book'
+require_relative 'rental'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def list_all_books
+    if @books.empty?
+      puts 'No Book found!!!'
+    else
+      @books.each { |book| puts "Title: #{book.title}, Author: #{book.author}" }
+    end
+  end
+
+  def list_all_people
+    if @people.empty?
+      puts 'No People found!!!'
+    else
+      @people.each do |person|
+        if person.is_a?(Student)
+          puts "Type: Student, ID: #{person.id}, Name: #{person.name}, Age: #{person.age}, Classroom: #{person.classroom.label}, Number of Rentals: #{person.rentals.count}"
+        else
+          puts "Type: Teacher, ID: #{person.id}, Name: #{person.name}, Age: #{person.age}, Number of Rentals: #{person.rentals.count}"
+        end
+      end
+    end
+  end
+
+  def create_person
+    print 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
+    choice = gets.chomp
+
+    print 'Age: '
+    age = gets.chomp
+
+    print 'Name: '
+    name = gets.chomp
+
+    case choice
+    when '1'
+      print 'Has parent permission? [Y/N]: '
+      permission = gets.chomp.downcase == 'y'
+
+      print 'Classroom: '
+      classroom = gets.chomp
```

Comment:
- [ ] It will be great to ensure that when you are creating a student, the application does not ask for the classroom. If you check the video that was presented to you, the application does not ask for the classroom. Kindly fix this